test/sanity/validate-modules/main.py

```diff
@@ -1091,7 +1091,7 @@ def _validate_docs(self):
     def _check_version_added(self, doc, existing_doc):
         version_added_raw = doc.get('version_added')
         try:
-            version_added = StrictVersion(str(doc.get('version_added', '0.0') or '0.0'))
+            version_added = StrictVersion(str(doc.get('version_added', '2.9') or '2.9'))
```

Comment:
Module PRs should not make changes to the module validator. Instead, revert this line and add `version_added` to the new module parameter (see my other comment).